docs/manual-release.md

```diff
@@ -34,14 +35,11 @@ In `frontend/iOS`  run `bundle exec fastlane match appstore` .
 - Use user [app-team@integreat-app.de](mailto:app-team@integreat-app.de) and password from `<passbolt - Digitalfabrik Fastlane Match>`
 - Set correct app_identifier (a list of identifier is also allowed): `app.sozialpass.nuernberg,de.nrw.it.ehrensachebayern`
 
-## Release Workflow
+## Create beta release
 
-- Create a release branch: `release-<version>`  (check out the next version in the google play console or in app store connect)
-- Create a  [Github release](https://github.com/digitalfabrik/entitlementcard/releases/new) based on that branch
-- Set a new tag with `<version>`
-- Generate release notes and update CHANGELOG.md
-- Update version in `/administration/package.json` and `frontend/pubspec.yaml`
+- Create a release branch on current main: `release-<version> f.e. "release-2024.11.1`  (check out the `version.json` in root folder and increment it)
```

Comment:
maybe you can link here to the "how to get the version section" you wrote in the cicd.md